.github/workflows/docker-image-saas.yml

```
@@ -0,0 +1,74 @@
+    runs-on: ubuntu-latest
+
+    env:
+      FULL_TAG: ${{ github.event.release.tag_name }}
+
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)